backend/apps/audit/management/commands/replay_events.py

```
@@ -0,0 +1,230 @@
+
+        # Separate many-to-many fields from snapshot
+        non_m2m_snapshot = {}
+        m2m_snapshot = {}
+        for field_name, value in snapshot.items():
+            try:
+                field = model_cls._meta.get_field(field_name)
+                if field.many_to_many:
+                    m2m_snapshot[field_name] = value
+                else:
+                    non_m2m_snapshot[field_name] = value
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)